packages/scoutgame/src/builderNfts/refreshEstimatedPayouts.ts

```diff
@@ -0,0 +1,93 @@
+import { prisma } from '@charmverse/core/prisma-client';
+import type { ISOWeek } from '@packages/dates/config';
+import { getCurrentSeasonStart } from '@packages/dates/utils';
+
+import { nftTypeMultipliers } from '../points/dividePointsBetweenBuilderAndScouts';
+import { getPointsCountForWeekWithNormalisation } from '../points/getPointsCountForWeekWithNormalisation';
+
+import type { BuilderNftWithOwners, SeasonNfts } from './getAllSeasonNftsWithOwners';
+import { getAllSeasonNftsWithOwners } from './getAllSeasonNftsWithOwners';
+
+/**
+ * Refresh estimated payouts for a given week
+ *
+ * @param builderToRefresh - Builder ID to refresh. If not provided, all builders will be refreshed
+ */
+export async function refreshEstimatedPayouts({
+  week,
+  builderIdToRefresh
+}: {
+  week: ISOWeek;
+  builderIdToRefresh?: string;
+}): Promise<SeasonNfts> {
+  const season = getCurrentSeasonStart(week);
+
+  const { normalisedBuilders } = await getPointsCountForWeekWithNormalisation({
```

Comment:
could be parallelized with the nfts query

Reply:
Fixed